.gitignore

```diff
@@ -34,3 +34,7 @@ local.properties
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 replay_pid*
+
+
+# Kotlin build directory introduced in K2, see https://kotlinlang.org/docs/whatsnew-eap.html#new-directory-for-kotlin-data-in-gradle-projects
+.kotlin
```

Comment:
Lol I saw this show up when I pushed the K2 change and I thought I was hallucinating, I should have checked if it was a new build dir. Thanks.